docs/introduction/vulkan_overview.md

```diff
@@ -9,58 +9,67 @@ parent: introduction
 
-## Multiplatform
+## Cross platform
 
-Vulkan is unique in its design compared to other GPU APIs because it has the same API for Mobile and Desktop GPUs. Due to that design, many features in Vulkan are optional and using them requires explicitly turning them on. It is not uncommon to have a feature which no Mobile GPU supports but is available on most desktop GPUs. In this tutorial, we are focusing on the Desktop side of things, so we can use features that wont directly run on smartphones and tablets. 
+Vulkan's design is unique compared to other Graphics APIs because it offers the same API for both Mobile and Desktop.
+This means that many features in Vulkan are optional and using them requires explicitly turning them on. 
+It is common to have features that are unsupported on Mobile but supported on most Desktop GPUs.
 
-If you wish to have a Vulkan application that will run on both Desktop and Mobile GPUs, its recommended to have two core render paths. As the API is the same, much of code can be shared, but the differences in features and fast-paths between the 2 targets means that having just one render code path will have suboptimal performance on at least one of the two platforms. 
+When running a Vulkan powered application on both Desktop and Mobile, you have the option of using one or two core render paths,
+it is recommended to run two core render path as the alternative will be at the cost of sub optimal performance.
+The API being unified means the two render paths can share code between them.
 
-During this guide we will explain many of the differences between the two types of GPUs.
+This tutorial will be focused on writing Vulkan for Desktop, however we will cover many of the differences between the two types of GPUs.
 
 ## Validation layers
 
-The Vulkan API is very large and rather easy to make mistakes with. Fortunately, when Vulkan was being designed, the creators realized that and made the Validation Layers to fix it. The Validation Layers is a bunch of error checking code that detects and reports 'invalid usage' of the API. They check for all kinds of errors such as using the wrong enums, threading violations, and object lifetimes. The layers are separate from the driver, and to work, they must intercept every function call, perform validation, then call the driver for you. The reason for that is when you want to turn off validation, you Vulkan calls go directly to the driver and can run as fast as possible.
+The vulkan API is vast and as a result it is easy to make mistakes, but this is where Validation Layers comes to the rescue.
+
+Validation Layers operate as a gateway between your function call and the graphics driver, this allows the Validation Layer to intercept all
+code being ran through Vulkan and validate that it is correct and then perform the call to the driver, this also means you will have reduced
```

Comment:
I do like the addition of how performance is impacted when running validation, but it should be noted why its slower. (though, fixing the previous comment I made about gateways would likely fix this one...)